### PR TITLE
Feature/step customize

### PR DESCRIPTION
**Pull request by Jeasmine**

### CODE REVIEW

File: Example/Example/Step.swift [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+class Step: UIView {
+    
+    @IBOutlet weak var stepImage: UIImageView!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+    }
+    
+    func addImage(image: UIImage) {
+        stepImage.image = UIImage(named: "abc1")
```
Comment (maxicasal) [inline Example/Example/Step.swift]: You can use a constant for image name here and also in ViewController.swift

File: Example/Example/ViewController.swift [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import StepControliOS
+
+class ViewController: UIViewController {
+    
+    @IBOutlet weak var stepControl: StepControl!
+    var steps: [UIView] = []
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.setupStep()
```
Comment (Jmaguerre) [inline Example/Example/ViewController.swift]: Remove unnecessary `self.` here and in any other place applicable

File: Example/Pods/StepControliOS/step-control-ios/DotTextLayer.swift [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+    let defaultFontSize: CGFloat = 14
+    let defaultForegroundColor = UIColor.white.cgColor
+    let defaultAlignmentMode = "center"
+
+    var isSelected = false
+    var isCompleted = false
+    var unselectedColor = UIColor.gray.cgColor
+    var selectedColor = UIColor.orange.cgColor
+    var completedColor = UIColor.orange.cgColor
```
Comment (maxicasal) [inline Example/Pods/StepControliOS/step-control-ios/DotTextLayer.swift]: Create a constant file and have:

struct Constants {
   struct Colors {
     static let selectedColor = ...
   }
}

File: Example/Pods/StepControliOS/step-control-ios/StepControl.swift [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+//
+
+import UIKit
+import QuartzCore
+
+class Steps: UIControl {
+
+    let minSteps = 2
+    let shrinkDotPercentage: CGFloat = 70.0
+    let marginBetweenDots: CGFloat = 40.0
```
Comment (maxicasal) [inline Example/Pods/StepControliOS/step-control-ios/StepControl.swift]: Move this to the Constants file too

File: Example/Pods/StepControliOS/step-control-ios/StepControl.swift [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+        var connectLayers = [CALayer]()
+
+        for index in 1...numberOfSteps {
+            let layer = DotTextLayer()
+            layer.selectedColor = self.selectedColor.cgColor
+            layer.unselectedColor = self.unselectedColor.cgColor
+            layer.completedColor = self.selectedColor.cgColor
+            layer.index = index
+            dotLayers.append(layer)
+        }
```
Comment (maxicasal) [inline Example/Pods/StepControliOS/step-control-ios/StepControl.swift]: dotLayers = createLayers()

File: Example/Pods/StepControliOS/step-control-ios/StepControl.swift [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+
+        for _ in 1...numberOfSteps-1 {
+            let connectLayer = CALayer()
+            connectLayer.backgroundColor = self.unselectedColor.cgColor
+            connectLayers.append(connectLayer)
+        }
```
Comment (maxicasal) [inline Example/Pods/StepControliOS/step-control-ios/StepControl.swift]: idem

**Pull request closed without merging**